src/components/DatePicker/DatePicker.js

```diff
@@ -236,9 +236,13 @@ export default class DatePicker extends Component {
       const onHook = (electedDates, dateStr, instance) => {
         this.updateClassNames(instance);
       };
+      const appendToNode =
+        typeof appendTo === 'string'
+          ? document.querySelector(appendTo) || document.body
```

Comment:
Can we remove `|| document.body` so that Flatpickr itself can handle that scenario?